### PR TITLE
DEVDOCS-3934: Rebranded Facebook

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3934](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3934)

## What changed?
Change Facebook to Meta

### CODE REVIEW

File: docs/api-docs/channels/channels-overview.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+BigCommerce's Channel Manager is the central place for merchants to discover, connect to, and manage their sales channels, including external channels like eBay, Amazon, Meta, and Instagram. These external sales channels can extend the merchant's control panel experience in a number of ways, such as pushing orders from these external channels into the control panel alongside their BigCommerce storefront orders, allowing them to be fulfilled in the same way, and managing products on these channel from within the control panel in many of the same ways as they do for their BigCommerce storefronts.
```
Comment (slsriehl) [inline docs/api-docs/channels/channels-overview.md:5]:
```suggestion
BigCommerce's Channel Manager is the central place for merchants to discover, connect to, and manage their sales channels, including external channels like eBay, Amazon, and Meta. These external sales channels can extend the merchant's control panel experience in a number of ways, such as pushing orders from these external channels into the control panel alongside their BigCommerce storefront orders, allowing them to be fulfilled in the same way, and managing products on these channel from within the control panel in many of the same ways as they do for their BigCommerce storefronts.
```

meta includes instagram

File: docs/api-docs/channels/channels-overview.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ Examples:
 
 - Amazon
 - eBay
-- Facebook Marketplace
+- Meta Marketplace
```
Comment (slsriehl) [inline docs/api-docs/channels/channels-overview.md:122]: let's verify that this is called `Meta Marketplace` for sure. I think they may have rebranded that.

File: docs/api-docs/channels/channels-overview.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ A platform on which merchants communicate their brand and products with their sh
 
 Examples:
 
-- Facebook Ads
+- Meta Ads
```
Comment (slsriehl) [inline docs/api-docs/channels/channels-overview.md:132]: let's verify that this is called `Meta Ads` for sure, from a branding POV.

File: docs/api-docs/channels/channels-overview.md [hunk 4]
```diff
@@ -119,7 +119,7 @@ Examples:
 
 - Amazon
 - eBay
-- Facebook Marketplace
+- Facebook by Meta Marketplace
```
Comment (slsriehl) [inline docs/api-docs/channels/channels-overview.md:122]:
```suggestion
- Meta for BigCommerce
```

File: docs/api-docs/channels/channels-overview.md [hunk 5]
```diff
@@ -129,11 +129,11 @@ A platform on which merchants communicate their brand and products with their sh
 
 Examples:
 
-- Facebook Ads
+- Facebook by Meta Ads
 - Google Shopping
 - WordPress Blog
 - Pinterest
-- Instagram
+- Facebook by Instagram
```
[the remark this answers is not in the document]
Comment (traci707) [reply docs/api-docs/channels/channels-overview.md]: This one remains the same per Gracie.

File: docs/api-docs/channels/channels-overview.md (hunk 4 above)
Comment (traci707) [reply docs/api-docs/channels/channels-overview.md]: This one remains the same per Gracie.

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**